tests/ResourceSyncingTest.php

```diff
@@ -598,6 +626,18 @@ public function getSyncedAttributeNames(): array
             'email',
         ];
     }
+
+    public function getCreateAttributeNames(): array
+    {
+        // attributes should be used when syncing resources from central to tenant DB
```

Comment:
Better writing:
```suggestion
        // Attributes used when syncing resources from central to tenant DB
```

Also, is this *really* what that method is? I don't think it necessarily has anything to do with central/tenant DB.

Reply:
Maybe better 

```suggestion
        // Attributes used when creating resources from central to tenant DB
```

As these attributes are used only when creating models from central to tenant. These does not used from syncing.